documentation/assemblies/assembly-iot-service-admin-guide.adoc

```diff
@@ -11,4 +11,6 @@ Service administrators typically install and configure the Internet of Things (I
 
 include::../modules/proc-iot-installing-services.adoc[leveloffset=+1]
 
+include::../modules/ref-iot-config-example.adoc[leveloffset=+1]
 
+include::../modules/ref-iot-config-example-adapters.adoc[leveloffset=+1]
```

Comment:
I would put all these under one section "IoT configuration examples". And then have two subsections: "Minimal IoT configuration example" and "IoT configuration example for tuning protocol adapters"

Reply:
@dejanb, good point, but I'd keep the files separate and create a new assembly called "Examples" that included both files.